softioc/cothread_dispatcher.py

```diff
@@ -40,3 +47,7 @@ def wrapper():
             if completion:
                 completion(*completion_args)
         self.__dispatcher(wrapper)
+
+    def __shutdown(self):
```

Comment:
There's no cleanup for the cothread dispatcher at the moment though, so this will never be called. Maybe put in a separate ticket for that though...

Reply:
I believe it does get called because of the atexit.register, I will test

Reply:
You're right, I missed that...